frame/bounties/src/lib.rs

```diff
@@ -223,6 +244,8 @@ pub mod pallet {
 		PendingPayout,
 		/// The bounties cannot be claimed/closed because it's still in the countdown period.
 		Premature,
+		/// The bounty cannot be closed because it has active child-bounties.
+		RequireNoActiveChildBounty,
```

Comment:
```suggestion
		ActiveChildBounty,
```

Error messages typically explain what _is wrong_, not what is _ought to be_.